### ISSUE TITLE
Does this work on localhost

### ISSUE THREAD

**Issue by Maxino22** [rmed/flask-discussion#1]
Hello , i just found out about this extention i have been trying to configure it but its not showing up on my templates

this is how i've set up

in my init.py
```
app.config['DISCUSSION_DISQUS_SHORTNAME'] = 'maxmuhanda'
discussion = Discussion(app)
```
and in my templates with the comment section this is what i have as 
```
{% import "flask_discussion/disqus.html" as disqus %}

   <div class="card-body d-flex flex-row">
            <html>
              <body>
          {{ disqus.render_comments(identifier="{{post.id}}", url="{{url_for('port.post', slug=post.slug)}}", title="{{post.title}}") }}
              </body>
          </html>
        </div>
```
no error and all it does is apply space on the area and nothing else... this is on my local host kindly assist me

**Comment by rmed** [comment]
Hello! As far as I remember (haven't used Disqus in a very long time) comments don't load in localhost environments, and I don't know if the Disqus documentation specifies anything regarding that matter.

**Comment by Maxino22** [comment]
Thank you and my last question with the way i have specified the parameters above is it okay will it work?

**Comment by rmed** [comment]
Yes, that should work. If you check the HTML rendered from the template you should be able to see the Disqus JS snippet.
